server/core/src/config.rs

```diff
@@ -0,0 +1,71 @@
+use confique::Config;
+use serde::Deserialize;
+use std::net::SocketAddr;
+use tracing::metadata::LevelFilter;
+
+/// Contains basic configuration entries.
+#[derive(Debug, Clone, Config)]
+pub struct AppConfig {
+    #[config(env = "SERVER_ADDRESS")]
+    pub server_address: SocketAddr,
+    #[config(env = "DATABASE_URL")]
+    pub database_url: String,
+    #[config(env = "COOKIE_KEY")]
+    pub cookie_key: String,
+    #[config(env = "LOG_LEVEL")]
+    pub log_level: LogLevel,
+    #[config(nested)]
+    pub mail_config: MailConfig,
+}
+
+/// Contains configuration tied to email sending.
+#[derive(Debug, Clone, Config)]
+pub struct MailConfig {
+    #[config(env = "SMTP_ADDRESS")]
+    pub smtp_address: String,
+    #[config(env = "SMTP_PORT")]
+    pub smtp_port: u16,
+    #[config(env = "SMTP_USERNAME")]
+    pub smtp_username: String,
+    #[config(env = "SMTP_PASSWORD")]
+    pub smtp_password: String,
+}
```

Comment:
Does something cry if you delete these /// comments ?
It is just noise, I am an outsider in this code, but it would be obvious that MailConfig is for mail config.

Reply:
Yep, it is unnecessary.